Tasks/ServiceFabricComposeDeployV0/ServiceFabricComposeDeploy.ps1

```diff
@@ -140,7 +140,7 @@ try
 
     if ($registryCredentials -ne "None")
     {
-        if (-not $isEncrypted -and $clusterConnectionParameters["ServerCertThumbprint"])
+        if (-not $isEncrypted -and $connectedServiceEndpoint.Auth.Parameters.CertLookUp -and ($clusterConnectionParameters["ServerCertThumbprint"] -or $clusterConnectionParameters["ServerCommonName"]))
```

Comment:
why is there a need to check for `CertLookUp` here? It will fail for older service connections which do not have this property

Reply:
Removed.